tdrs-frontend/.nvmrc

```diff
@@ -0,0 +1 @@
+12.18
```

Comment:
**TLDR:**

- Open to changing the semantic version specificity choice - it was just an implementation opinion at the time. 
- [trussworks/react-uswds](https://github.com/trussworks/react-uswds) doesn't play nice with latest node at this time.

**Node Version:**

I wanted to do at least `14` .. I opted not to do `lts` because when `lts` changes one day, there might be not-insignificant breaking changes -- it doesn't seem we would want to automatically jump major versions, but make it a conscious choice. 

The reason the minor version was chosen (`.18`) was that even minor version increases can cause breaking changes. In favor of being explicit and conscious about updates, the decision was made to specify MAJOR and MINOR, but let PATCH update as needed, since usually the PATCH level involves minor security updates and bug fixes, which we would want. 

These two posts summarize the premise:
- [https://cloudnativedevopsblog.com/posts/docker-tags/](https://cloudnativedevopsblog.com/posts/docker-tags/#:~:text=Here%20is%20a%20quick%20summary%20of%20the%20pros/cons%20of%20each:)
- [https://derickbailey.com/2017/03/09/selecting-a-node-js-image-for-docker/](https://derickbailey.com/2017/03/09/selecting-a-node-js-image-for-docker/#:~:text=Select%20Your%20Node.js%20Version)

Open to changing. Thoughts? 

**Why 12?**:

The reason it's node `12` is that [trussworks/react-uswds](https://github.com/trussworks/react-uswds) doesn't support and has a conflict with 14. 

To reproduce:

```
adamcaron ~/raft/tdrs/tdrs-frontend nvm use system
Now using system version of node: v14.5.0 (npm v6.14.5)

adamcaron ~/raft/tdrs/tdrs-frontend yarn add react-uswds
yarn add v1.22.4
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error @trussworks/react-uswds@1.8.0: The engine "node" is incompatible with this module. Expected version "10.x - 12.x". Got "14.5.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

![image](https://user-images.githubusercontent.com/7072066/89789581-c75c2d80-daee-11ea-9729-8ccf828d6d27.png)

The [trussworks project](https://github.com/trussworks/react-uswds) seems well maintained and I can only assume they will provide support for this.